From 6fbbd62879d4f8b2126a0421b2617b2195568629 Mon Sep 17 00:00:00 2001
From: mkg20001 <mkg20001@gmail.com>
Date: Mon, 2 Jan 2017 13:09:09 +0100
Subject: [PATCH] Fix some errors

---
 src/Ui/UiRequest.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Ui/UiRequest.py b/src/Ui/UiRequest.py
index 560486e..6d2ac9b 100644
--- a/src/Ui/UiRequest.py
+++ b/src/Ui/UiRequest.py
@@ -336,10 +336,10 @@ class UiRequest(object):
             self.server.wrapper_nonces.remove(self.get["wrapper_nonce"])
 
         referer = self.env.get("HTTP_REFERER")
-        if referer and path_parts:  # Only allow same site to receive media
-            if not self.isMediaRequestAllowed(path_parts["request_address"], referer):
-                self.log.error("Media referrer error: %s not allowed from %s" % (path_parts["address"], referer))
-                return self.error403("Media referrer error")  # Referrer not starts same address as requested path
+        #if referer and path_parts:  # Only allow same site to receive media
+        #    if not self.isMediaRequestAllowed(path_parts["request_address"], referer):
+        #        self.log.error("Media referrer error: %s not allowed from %s" % (path_parts["address"], referer))
+        #        return self.error403("Media referrer error")  # Referrer not starts same address as requested path
 
         if path_parts:  # Looks like a valid path
             address = path_parts["address"]
-- 
2.7.4

